src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -891,3 +891,10 @@
     </datalist>
   {% endif %}
 {% endblock text_widget %}
+
+{%- block number_widget -%}
```

Comment:
Already defined in this file and what about `form_div_layout.html.twig` file?

Reply:
This form uses the new theme new theme, I believe, overrides the form_div_layout. I don't need to override all of the widgets but I need to override this one to add help_block.

Reply:
But I removed this line anyway. It was already done in another PR and that one was merged before.